src/me/drton/jmavsim/KinematicObject.java

```diff
@@ -36,16 +36,18 @@
     protected TransformGroup transformGroup;
     protected BranchGroup branchGroup;
 
-    public KinematicObject(World world) {
+    public KinematicObject(World world, boolean gui) {
```

Comment:
I would prefer a clearer name for `gui` like `withGui` or `showGui`.

Reply:
I'll change it to showGui